src/calendar.js

```diff
@@ -58,6 +67,36 @@ const Day = React.createClass({
 const Calendar = React.createClass({
   displayName: 'Calendar',
 
+  getDate(i, w) {
+    const prevMonth = w === 0 && i > 7
```

Comment:
Probably should pull out this logic into a quick getter since we do it 3 times

Reply:
Done